app/src/main/java/me/magnum/melonds/utils/Window.kt

```diff
@@ -0,0 +1,8 @@
+package me.magnum.melonds.utils
```

Comment:
Since this file contains extensions, it would be better suited to be placed in the `me.magnum.melonds.extensions` package, with the file name `WindowExtensions`.

Reply:
Sure!